docs/content/Cube-Cloud/Configuration/VPC/Connecting-with-a-VPC-Azure.mdx

```diff
@@ -5,17 +5,56 @@ permalink: /cloud/configuration/connecting-with-a-vpc/azure
 
 ## Prerequisites
 
-To allow Cube Cloud to connect to a VPC on Azure, the following information is
-required:
+To allow Cube Cloud to connect to a Virtual Network on Azure, the following information is required:
+- **Virtual Network Name:** You can find it at `Virtual Networks` section on [Azure Portal][https://portal.azure.com].
+- **Tenant ID:** You can find it at `Azure Active Directory` > `Properties` > `Tenant ID` on [Azure Portal][https://portal.azure.com].
 
 ## Setup
 
-### VPC Peering
+Unfortunately Azure cross-tenant peering suppose that you grant peering role to the user id/service principal to the peering party
```

Comment:
Not really sure what this sentence means 😅

Reply:
It is a shout-out to the worst user flow in the industry